lib/constructs/interface.js

```diff
@@ -365,18 +365,15 @@ Interface.prototype.generateToString = function () {
     }
   }
   if (!hasToString) {
```

Comment:
This condition is now flipped; it should be `if (hasToString)`.

I'll fix this before merging.